app/queries/my_facilities/blood_pressure_control_query.rb

```diff
@@ -61,6 +61,7 @@ def overall_controlled_bps
     @overall_controlled_bps ||=
       LatestBloodPressuresPerPatient
       .where(registration_facility: facilities)
+      .where('patient_recorded_at < ?', Time.current.beginning_of_day - REGISTRATION_BUFFER)
```

Comment:
What's the corresponding denominator method? We should make this method something like

```
def overall_controlled_bps
  denominator.under_control
end
```

to ensure that the numerator is always a subset of the denominator as intended.

Reply:
The denominator is `overall_patients` which is now a query on `Patient` instead of `LatestBloodPressurePerPatient`.
We had something along the lines of what you recommended earlier, but lost it when we changed the denominator to be a query on `Patients` directly.
I've updated this condition to be `.where(patients: overall_patients)` instead.